feat: ROOT-92: Hide unused columns in DM setting

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/17445606011)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/562
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17445606011

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17472483817)
> Successfully merged:  create mode 100644 web/libs/editor/src/tags/POCUI.jsx

jombooth committed:
Merge branch 'develop' into 'fb-ROOT-92'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17472483817

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17475277821)
> Successfully merged: Already up to date.

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17477976473)
> Successfully merged:  1 file changed, 9 insertions(+), 3 deletions(-)

jombooth committed:
Merge branch 'develop' into 'fb-ROOT-92'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17477976473

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/17479079350
